docs/source/markdown/podman-stats.1.md

```diff
@@ -32,15 +32,24 @@ Valid placeholders for the Go template are listed below:
 
 | **Placeholder** | **Description**    |
 | --------------- | ------------------ |
-| .ID             | Container ID       |
```

Comment:
`podman stats --format  {{.ID}}` works, I assume my completion code has a big somewhere and does not catch it for some reason.

Reply:
It's much weirder than that. I seriously don't understand this command: the man page documents `CPUPerc`, which is not in `--format`. `--format` lists something that _looks_ similar, `PerCPU`, but `PerCPU` is actually useless, it does not emit anything informative (just empty brackets). I think the man page here is mostly correct, but have not been able to understand why there are so many differences. Maybe some sort of aliasing mechanism?

Reply:
PerCPU is a valid field on the struct: https://github.com/containers/podman/blob/c379014ee4e57dc19669ae92f45f8e4c0814295b/libpod/define/containerstate.go#L137

Reply:
Yes, but it doesn't give anything useful, and is very definitely *not* the same as `CPUPerc`, which is not listed by autocomplete. This is way above anything I can understand.

Reply:
https://github.com/containers/podman/pull/14216 should fix this bug

Reply:
It sure seems to, and brings to light a number of other fields (in `podman events` and `podman images`) that I hadn't noticed. Thank you!